### PR TITLE
Do not render constraint for un configured field.

### LINKED ISSUE #168: Range limit constraint does not take into account blacklight config
Our blacklight application has two separate blacklight range fields that are not part of the same configuration or even on the same controller.  However when we switch between the controller paths the application will render all the range limit param constraints even if that controller should not know about all the range limit fields that are part of the query parameters.

### PR DESCRIPTION
**Pull request by dkinzer**
Fixes #168

Fixes issue where a constraint gets rendered even though the controller
should not know anything about the particular field whose constraint is
being rendered.

### CODE REVIEW

File: lib/blacklight_range_limit/view_helper_override.rb [hunk 1]
```diff
@@ -89,8 +89,10 @@ def range_config(solr_field)
     def range_params(my_params = params)
       return {} unless my_params[:range].is_a?(ActionController::Parameters) || my_params[:range].is_a?(Hash)
 
+      facet_field_keys = blacklight_config.facet_fields.keys
       my_params[:range].select do |_solr_field, range_options|
```
Comment (cbeer) [inline lib/blacklight_range_limit/view_helper_override.rb:93]: By convention, now that `_solr_field` is used, we should drop the `_`. I also don't think we need to memoize the list of keys, right?

```suggestion
      my_params[:range].select do |solr_field, range_options|
        next unless range_options && blacklight_config.facet_fields[solr_field].present?
```

And, fwiw, this should be the behavior once we get range limit using components too.

Comment (dkinzer) [reply lib/blacklight_range_limit/view_helper_override.rb:93]: Yeah, that makes total sense.  Thanks.

**Comment by dkinzer** [comment]
This doesn't look like it no longer applies to the main branch.  I'm closing and reopening to apply to the 7.x-release branch.

**Comment by dkinzer** [comment]
I'm putting this back into draft until I get a chance to see if the latest version fixes the issue I was having.

**Comment by dkinzer** [comment]
The latest version of blacklight_range_limit does not have this issue.

**Pull request closed without merging**